src/main/webapp/components/search-forms/route.tsx

```diff
@@ -106,6 +109,12 @@ const SearchForm = (props: SearchFormProps) => {
               onDelete={props.onDelete}
               isWritable={props.canWrite}
             />
+            {typeof props.toggleDefaultForm === 'function' && (
+              <DefaultFormInteraction
```

Comment:
:pencil2: Could you instead change what DefaultFormInteraction renders based on the props you pass it?